src/NumericalAlgorithms/Spectral/CMakeLists.txt

```diff
@@ -28,7 +28,12 @@ target_link_libraries(
   ErrorHandling
   Libsharp
 
+  INTERFACE
+  # LinearOperators should be PUBLIC but there is a cyclic dependency
+  LinearOperators
```

Comment:
should't it be the other way round? `LinearOperators` should depend on `Spectral`, not `Spectral` on `LinearOperators`

Reply:
Yes, that's the comment on liken 32, and in the comment on the PR I said I didn't fix the cyclic dependencies in this PR